src/components/macroview/rightPanel/editForms/KeyPressForm.tsx

```diff
@@ -81,13 +105,51 @@ export default function KeyPressForm({
     [selectedElement, selectedElementId, updateElement]
   )
 
+  useEffect(() => {
+    if (resetTriggered) {
+      toast({
+        title: 'Default duration applied',
+        description: `Applied default duration of ${DefaultMacroDelay}ms`,
+        status: 'info',
+        duration: 4000,
+        isClosable: true
+      })
+      onInputBlur()
+      setResetTriggered(false)
+    }
+  }, [onInputBlur, resetTriggered, toast])
+
+  const onResetClick = () => {
+    setKeypressDuration(DefaultMacroDelay)
+    setResetTriggered(true)
+  }
```

Comment:
The resetTriggered state variable is kinda redundant? Why don't you just trigger the toast inside of the `onResetClick` function?

Reply:
Tweaked here: https://github.com/WootingKb/wooting-macros/pull/194/commits/f80d476a0706253f4164e241766865ef35ef95bd

and: 

[4f582e9](https://github.com/WootingKb/wooting-macros/pull/194/commits/4f582e99ad655dc65d498bdaebf3e2248711d34a)